### ISSUE TITLE
When I have two mysql data sources or Multiple data source，how to use lapis

### ISSUE THREAD

**Issue by sixtool** [leafo/lapis#649]
When I have two mysql data sources or Multiple data source，how to use lapis

**Comment by leafo** [comment]
There's no easy way to do this right now since lapis manages a singleton for the db object (the primary database connection). You're welcome to create your own modules for any other connections you want to have open. Are you trying to use models along with your secondary database? If so, it's possible to provide an alternate `db` object on the class level for models. Tell me if you need more info.

**Comment by tanwenliang** [comment]
Yes!,  I trying to use models along the secondary database,How to use it?
 oh, I can only create my own module to connection database ?

**Comment by tanwenliang** [comment]
`local config = require("lapis.config").get()
if config.postgres then
  return require("lapis.db.postgres")
elseif config.mysql then
  return require("lapis.db.mysql")
elseif config.masql2 then        --  like this? 
  return require("lapis.db.mysql")   -- 
else
  return error("You have to configure either postgres or mysql")
end`

**Comment by edubart** [comment]
I'm also needing this. In my case I have two mysql instances on different servers, so I do need two connections using the resty_mysql backend.

**Comment by tanwenliang** [comment]
@edubart  thks, i try it 
this good idea

**Comment by emadsamir** [comment]
laefo, any update on a good approach for this with the latest version?
I personally think, being able to provide the connection options from within the model would be a good approach. Is that what you meant by "provide an alternate db object on the class level for models"
